src/security/builtin_plugins/authentication/src/authentication.c

```diff
@@ -199,14 +204,13 @@ typedef struct HandshakeInfo {
 
 typedef struct dds_security_authentication_impl {
     dds_security_authentication base;
+    int id; //sample internal member
```

Comment:
What is this?

Reply:
I do not know. The id is not used therefor I will remove the it.